### PR TITLE
Create win_susp_ntlm_brute_force.yml

### PR DESCRIPTION
**Pull request by JSHOX1**

### CODE REVIEW

File: rules/windows/builtin/ntlm/win_susp_ntlm_brute_force.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    - attack.t1110
+logsource:
+    product: windows
+    service: ntlm
+    definition: Requires events from Microsoft-Windows-NTLM/Operational
+detection:
+    selection:
+        EventID: 8004
+    devicename:
+        Workstation_name:
```
Comment (frack113) [inline rules/windows/builtin/ntlm/win_susp_ntlm_brute_force.yml:19]: From shema:
`<data name="WorkstationName" inType="win:UnicodeString" outType="xs:string"/>`

File: rules/windows/builtin/ntlm/win_susp_ntlm_brute_force.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            - 'Rdesktop'
+            - 'Remmina'
+            - 'Freerdp'
+            - 'Windows7'
+            - 'Windows8'
+            - 'Windows2012'
+            - 'Windows2016'
+            - 'Windows2019'
+    condition: selection and 1 of devicename
```
Comment (frack113) [inline rules/windows/builtin/ntlm/win_susp_ntlm_brute_force.yml:28]: `condition: selection and devicename`
you will get the query :  eventid:8004 AND (*Rdesktop* OR *Remmina* OR...OR *Windows2019*)

File: rules/windows/builtin/ntlm/win_susp_ntlm_brute_force.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+title: NTLM Brute Force
+status: experimental
```
Comment (frack113) [inline rules/windows/builtin/ntlm/win_susp_ntlm_brute_force.yml:2]: `id: 9c8acf1a-cbf9-4db6-b63c-74baabe03e59`
I took one from https://www.uuidgenerator.net/version4

Comment (JSHOX1) [reply rules/windows/builtin/ntlm/win_susp_ntlm_brute_force.yml:3]: Thanks @frack113 I have made the change.

**Pull request merged**